packages/expo/src/Expo.fx.ts

```diff
@@ -2,3 +2,13 @@ import './environment/react-native-logs.fx';
 
 // load expo-asset immediately to set a custom `source` transformer in React Native
 import 'expo-asset';
+
+import { NativeModulesProxy, Platform } from '@unimodules/core';
+if (NativeModulesProxy.ExpoUpdates?.shouldShowNoRuntimeVersionWarning) {
+  console.warn(
```

Comment:
What if we threw a JS error here to make it more visible? This way it would be immediately visible in production apps too.

Reply:
maybe warn if `__DEV__` and throw error otherwise? i think the warning would be annoying enough in dev for people to fix it, without forcing them to interrupt whatever they're currently trying to do

Reply:
That seems good.